tools/cpp/unix_cc_toolchain_config.bzl

```diff
@@ -285,6 +292,17 @@ def _impl(ctx):
                 with_features = [with_feature_set(features = ["opt"])],
             ),
         ],
+        env_sets = [
+            env_set(
+                actions = all_link_actions + lto_index_actions + [ACTION_NAMES.cpp_link_static_library],
+                env_entries = ([
+                    env_entry(
+                        key = "ZERO_AR_DATE",
```

Comment:
This is required to get reproducible archives / links on macOS, it shouldn't hurt applying to other platforms